pkg/apis/clouddns/v1/record_types.go

```diff
@@ -0,0 +1,14 @@
+package v1
+
+// anxcloud:object:hooks=ResponseDecodeHook,PaginationSupportHook
+
+type Record struct {
+	Identifier string `json:"identifier,omitempty" anxcloud:"identifier"`
+	ZoneName   string
```

Comment:
no json tag, do you mean not to encode this?
it needs a `json:"-"` tag then

Reply:
Indeed